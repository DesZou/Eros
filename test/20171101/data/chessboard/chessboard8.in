8
oooooooo
*ooooooo
oooooo*o
oo*oooo*
oo*ooooo
ooo**ooo
oooooooo
ooooooo*