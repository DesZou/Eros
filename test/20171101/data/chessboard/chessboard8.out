1
445910711
8877452
78736621
0
0
0
0
0
